pyglmnet/pyglmnet.py

```diff
@@ -530,7 +534,7 @@ class GLM(BaseEstimator):
         https://core.ac.uk/download/files/153/6287975.pdf
     """
 
-    def __init__(self, distr='poisson', alpha=0.5,
+    def __init__(self, distr='binomial', alpha=0.5,
```

Comment:
oops, the default value changed here.

Reply:
Yes, somehow it fails sklearn with `check_estimator` due to `predict_proba`. I didn't resolve it here.

Reply:
humm ... I see. Let's bring it back to the default and see if we can fix it? Is that the only thing left that didn't work?

Reply:
Yes, will do bring it back to the default!